wire-ios-sync-engine/Source/SessionManager/SessionManager.swift

```diff
@@ -244,6 +244,8 @@ public final class SessionManager: NSObject, SessionManagerType {
     let configuration: SessionManagerConfiguration
     var pendingURLAction: URLAction?
     let apiMigrationManager: APIMigrationManager
+    var cryptoboxMigrationManager: CryptoboxMigrationManagerInterface = CryptoboxMigrationManager()
```

Comment:
Seems like `cryptoboxMigrationManager` is not used in this file. Is this intended?

Reply:
nice catch